orb-java-core/src/test/kotlin/com/withorb/api/services/blocking/WebhookServiceTest.kt

```diff
@@ -0,0 +1,186 @@
+// File generated from our OpenAPI spec by Stainless.
+
+package com.withorb.api.services.blocking
+
+import com.withorb.api.TestServerExtension
+import com.withorb.api.client.okhttp.OrbOkHttpClient
+import com.withorb.api.core.http.Headers
+import com.withorb.api.errors.OrbException
+import com.withorb.api.models.*
+import java.time.Clock
+import java.time.Instant
+import java.time.LocalDateTime
+import java.time.ZoneOffset
+import java.time.format.DateTimeFormatter
+import org.assertj.core.api.Assertions.*
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+
+@ExtendWith(TestServerExtension::class)
```

Comment:
```suggestion
```

I think this isn't needed because we're not actually making requests in this test